poi-ooxml/src/main/java/org/apache/poi/openxml4j/opc/PackageRelationshipCollection.java

```diff
@@ -373,6 +373,14 @@ public PackageRelationshipCollection getRelationships(String typeFilter) {
         return relationshipsByID.values().iterator();
     }
 
+    /**
+     * Get this collection's spliterator.
+     */
```

Comment:
could you add `@since POI 5.2.0` on any new public methods?

Reply:
Sure

Reply:
Done